src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { Loader } from './Loader';
+import { TablePage } from './TablePage';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isLoadingError, setIsLoadingError] = useState(false);
+
+  useEffect(() => {
+    getPeople()
+      .then(setPeople)
+      .catch(() => setIsLoadingError(true))
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="box table-container">
+          {isLoading ? (
+            <Loader />
+          ) : (
+            <>
+              {isLoadingError ? (
+                <p
+                  data-cy="peopleLoadingError"
+                  className="has-text-danger"
+                >
+                  Something went wrong
+                </p>
+              ) : (
+                <>
+                  {!isLoading && people.length > 0 ? (
+                    <TablePage people={people} />
+                  ) : (
+                    <p data-cy="noPeopleMessage">
+                      There are no people on the server
+                    </p>
+                  )}
+                </>
+              )}
+            </>
+          )}
```

Comment:
Still not fixed. Do not use ternary nesting

![2023-09-12_11h15_51](https://github.com/mate-academy/react_people-table-basics/assets/61904995/ea6b31c0-8e20-4fc2-a62f-38c0e6a40b07)